.storybook/preview.js

```diff
@@ -182,6 +182,8 @@ window.me = {
 window._ = _
 window._.string = require('underscore.string')
 
+window.moment = require('moment')
```

Comment:
could this be avoided? I guess it wouldn't change the bundle with moment already there but moment is hugeeee